projects/Mallard/ios/Mallard.xcodeproj/project.pbxproj

```diff
@@ -995,6 +1019,8 @@
 				6711322A26474BF5BF399BBB /* libRNKeychain.a */,
 				0DEEC74B0C31419DA23DA0F4 /* libReactNativeConfig.a */,
 				8490E7B295514AB79CA53274 /* libReactNativeConfig-tvOS.a */,
+				8BD0D6D6E6324CEFAC2F5167 /* libRNSentry.a */,
```

Comment:
I cant read this file - just the sentry library names seem to not be related to the PR?

Reply:
I agree, they just seemed to come through and it felt a bit odd not to include them. I think because I had to add `sentry.properties` for the local build?